R/as_gt.R

```diff
@@ -242,115 +250,24 @@ as_gt.gs_design <- function(
     display_inf_bound = FALSE,
     ...) {
 
-  method <- intersect(class(x), c("ahr", "wlr", "combo", "rd"))[1]
-  full_alpha <- attr(x, "full_alpha")
```

Comment:
@LittleBeannie I have a question regarding a difference between `as_gt()` and `as_rtf()` that I noticed while comparing the two: for `as_gt()`, we obtain `full_alpha` from the attribute of `x`, whereas for `as_rtf()`, `full_alpha` is its argument and not retrieved from `x`'s attributes. Is this expected? I'm not sure if you or @elong0527 is the right person to ask.

Reply:
Hi @yihui , yes, I can answer the question. The `full_alpha` should be retrieved from `x`'s attributes. I will prepare a PR to get it there.

Reply:
If you want to do it, please wait until this PR is merged, or I can just do it in this PR.

Reply:
Thanks for the confirmation. I know there are many changes occurring in this pull request, so let's wait until this PR is merged.